src/App.tsx

```diff
@@ -48,10 +46,13 @@ function App() {
     }
     logseq.Editor.registerSlashCommand("ollama", ollamaUI)
     logseq.Editor.registerBlockContextMenuItem("Ollama: Create a flash card", convertToFlashCardFromEvent)
-    logseq.Editor.registerBlockContextMenuItem("Ollama: Summarize block", summarizeBlockFromEvent)
     logseq.Editor.registerBlockContextMenuItem("Ollama: Divide into subtasks", DivideTaskIntoSubTasksFromEvent)
-    logseq.Editor.registerBlockContextMenuItem("Ollama: Prompt from Block", promptFromBlockEvent)
-    logseq.Editor.registerBlockContextMenuItem("Ollama: Expand Block", expandBlockEvent)
+    logseq.Editor.registerBlockContextMenuItem("Ollama: Prompt from Block", promptFromBlockEventClosure())
+    logseq.Editor.registerBlockContextMenuItem("Ollama: Summarize block", promptFromBlockEventClosure("Summarize: "))
+    logseq.Editor.registerBlockContextMenuItem("Ollama: 总结 Block", promptFromBlockEventClosure("总结: "))
```

Comment:
While I love to support all languages I think let's stick with only English and not clutter the UI, Though in the future we could support localization of all commands

Reply:
Definitely, I am trying to figure out how to make these in configuration instead of hardcoding it. The ideal place would be in the settings but it doesn't seem the Logseq setting is that flexible.

Reply:
I am thinking of using a configuration Page “Ollama Configuration ” in which each block represents a command. I am going to give this a try this week. Given the verbose nature of the LLM commands and Logseq's text editing, this may be a good approach.

Reply:
> I think we can allow this through reading some kind of config file which can open a lot of possibilities like custom prompts instead of the default ones, but I don't know if it's against logseq policy that plugins can read user files

I just update the pull request to include a configuration page. Each block represents a new command available. 


![image](https://github.com/omagdy7/ollama-logseq/assets/6722/53578ab5-62c0-46cf-90f2-041b27d65188)
![image](https://github.com/omagdy7/ollama-logseq/assets/6722/7b382963-4351-4d14-a65d-6819b0abfa03)